src/Aspire.Dashboard/Extensions/FluentUIExtensions.cs

```diff
@@ -25,4 +25,20 @@ internal static class FluentUIExtensions
 
         return attributes;
     }
+
+    public static Dictionary<string, object> GetOpenTextVisualizerAdditionalAttributes(string textValue, string textValueDescription, params (string Attribute, object Value)[] additionalAttributes)
+    {
+        var attributes = new Dictionary<string, object>
+        {
+            { "data-textvisualizer-text", textValue },
```

Comment:
The copy button and the text visualizer button have separate attributes containing the text value. In case the data is very large, I'd like to combine them to one shared value.

Reply:
Changed to data-text